frontend/src/modules/dashboard/components/dashboard-count.vue

```diff
@@ -162,30 +166,21 @@ export default {
       const currentScore = this.computedScore(current)
       const previousScore = this.computedScore(previous)
       const diff = currentScore - previousScore
-      if (this.percentage) {
-        if (previousScore === 0) {
-          if (currentScore === 0) {
-            return '='
-          }
-          return `+100%`
-        }
-        if (diff > 0) {
-          return `+${Math.round(
-            (diff / previousScore) * 100
-          )}%`
-        }
-        if (diff < 0) {
-          return `${Math.round(
-            (diff / previousScore) * 100
-          )}%`
-        }
-      } else {
-        if (diff > 0) {
-          return `+${diff}`
-        }
-        if (diff < 0) {
-          return diff
+      if (previousScore === 0) {
+        if (currentScore === 0) {
+          return '='
         }
+        return `+100% (${currentScore})`
+      }
+      if (diff > 0) {
+        return `${Math.round(
+          (diff / previousScore) * 100
+        )}% (${diff})`
+      }
+      if (diff < 0) {
+        return `${Math.round(
```

Comment:
Both in positive and negative growth we shouldn't show the `-` or `+` sign. Nuno warned me about this as well in reports. For both scenarios this should be UI:
<img width="164" alt="Screenshot 2023-02-22 at 15 35 59" src="https://user-images.githubusercontent.com/20134207/220672993-a8725849-144e-44c3-903f-a84f043699da.png">
<img width="161" alt="Screenshot 2023-02-22 at 15 36 04" src="https://user-images.githubusercontent.com/20134207/220673020-4b7aaa11-8817-49af-8825-e54b94ce0bb2.png">